packages/paste-website/src/pages/components/separator/index.mdx

```diff
@@ -0,0 +1,182 @@
+
+<LivePreview scope={{Box, Separator}} language="jsx">
+  {`<Box backgroundColor="colorBackgroundBody" padding="space50">
+  <Separator orientation="horizontal" verticalSpacing="space50" />
+</Box>`}
+</LivePreview>
+
+### Vertical separator
+
+Horizontal separation can be created by using a vertical separator. This is achieved by setting `orientation="vertical"`. A `horizontalSpacing` styling property is provided to let you to create space on either side of the separator. `horizontalSpacing` will take any [Spacing Token](/tokens#spacings) as its value.
+
+<LivePreview scope={{Separator, Text, Box}} language="jsx">
+  {`<Box backgroundColor="colorBackgroundBody" padding="space50" display="flex">
+  <Text as="span">Option 1</Text>
+  <Separator orientation="vertical" horizontalSpacing="space40" />
+  <Text as="span">Option 2</Text>
+  <Separator orientation="vertical" horizontalSpacing="space40" />
+  <Text as="span">Option 3</Text>
+  <Separator orientation="vertical" horizontalSpacing="space40" />
+  <Text as="span">Option 4</Text>
+  <Separator orientation="vertical" horizontalSpacing="space40" />
+  <Text as="span">Option 5</Text>
+</Box>`}
+</LivePreview>
+
+### Spacing
+
+The Separator comes with 2 spacing properties: `horizontalSpacing` and `verticalSpacing`. You can use these properties to create space between the Separator and the two pieces of UI it sits between.
+
+When used, the spacing properties create equal space in both directions along the orientation you choose.
+
+#### Responsive spacing
+
+`horizontalSpacing` and `verticalSpacing` are responsive props. You can provide them an array of spacing values to change the spacing based on the screen sizes supported by your [theme](/tokens/theme-package#theme-provider).
+
+<LivePreview scope={{Separator, Text, Box}} language="jsx">
+  {`<Box backgroundColor="colorBackgroundBody" padding="space50">
+  <Box display="flex">
+    <Text as="span">Option 1</Text>
+    <Separator orientation="vertical" horizontalSpacing="space80" />
+    <Text as="span">Option 2</Text>
+  </Box>
+  <Separator orientation="horizontal" verticalSpacing="space80" />
+  <Box display="flex">
+    <Text as="span">Option 1</Text>
+    <Separator orientation="vertical" horizontalSpacing="space80" />
+    <Text as="span">Option 2</Text>
+  </Box>
+</Box>`}
+</LivePreview>
+
+## When to use the Separator
+
+### Page separation
+
+A Separator can be used to create visual separation between two parts of a page or within page content. It can also be useful in creating hierarchy within a page.
+
+The most common use case closely reflects the HTML `<hr />` (horizontal rule) element. The HTML specification describes a horizontal rule as representing a "thematic break" within a section of a page. It is used to inject separation between two elements. You can use it as a transition to another topic or represent that 2 things in the same section are not directly related.
```

Comment:
"It is used to inject separation between two elements" could prob be removed since it's basically repeating the paragraph before.